### ISSUE TITLE
Add key to `remember` of Compose-ViewModel

### ISSUE THREAD

**Issue by RinOrz** [InsertKoinIO/koin#1081]
At present, when the composable is recompose, the **ViewModel** instance of a different qualifier will not be fetch. The `qualifier` and `parameters` should be used as the `remember` key here.

https://github.com/InsertKoinIO/koin/blob/master/android-compose/koin-androidx-compose/src/main/java/org/koin/androidx/compose/ViewModelComposeExt.kt#L44

**Comment by arnaudgiuliani** [comment]
> should be used as the remember key here.

you mean we need to "remember" the qualifier & parameters?

**Comment by RinOrz** [comment]
Yeah, so:
```kotlin
@Composable
inline fun <reified T : ViewModel> getViewModel(
  qualifier: Qualifier? = null,
  noinline parameters: ParametersDefinition? = null,
): T {
  val owner = LocalViewModelStoreOwner.current
  return remember(qualifier, parameters) {
    owner.getViewModel(qualifier, parameters)
  }
}
```

**Comment by arnaudgiuliani** [comment]
Thanks

**Comment by arnaudgiuliani** [comment]
Fixed in `cbb25e98e1dec4b99f44a15615bb69629ee076d7`
